test/easyconfigs/easyconfigs.py

```diff
@@ -428,10 +428,16 @@ def check_python_packages(self, changed_ecs):
     def check_sanity_check_paths(self, changed_ecs):
         """Make sure a custom sanity_check_paths value is specified for easyconfigs that use a generic easyblock."""
 
+        # PythonBundle already has a decent customised sanity_check_paths for bundles of Python packages
+        whitelist = ['PythonBundle']
```

Comment:
`PythonPackage` also has a customized `sanity_check_paths`
isn't it better to check if the `sanity_check_paths` is equal to the default instead of having to maintain a whitelist?

Reply:
Good point on `PythonPackage`, but checking against the default value is tricky because of how `PythonBundle` & `PythonPackage` add the default, see https://github.com/easybuilders/easybuild-easyblocks/blob/master/easybuild/easyblocks/generic/pythonbundle.py#L110